packages/tfchain_client/src/contracts.ts

```diff
@@ -289,6 +289,19 @@ class Contracts extends QueryContracts {
     });
   }
 
+  @checkConnection
+  async cancelCollective(options: CancelOptions) {
+    const contract = await this.get(options);
+    if (!contract) {
+      return;
+    }

```

Comment:
Do you think we need to catch and return the error if there is an error, I'm thinking in case of failure